lib/counter_culture/extensions.rb

```diff
@@ -33,6 +33,7 @@ def counter_culture(relation, options = {})
         # add the counter to our collection
         @after_commit_counter_cache << Counter.new(self, relation, options)
       end
+      alias_method :cache_count_on, :counter_culture
```

Comment:
sorry I missed that. That was my own semantic renaming of the method. I was leaving that for a latter PR. I will fix the PR.